html/browsers/windows/document-domain-removed-iframe.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>document.domain and removed iframe interaction</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<!-- This is a test for https://crbug.com/1095145 where window
+     properties become undefined for document.domain-using removed
+     iframes -->
+
+<div id="log"></div>
+
+<script>
+"use strict";
+
+promise_test(t => {
+  return new Promise(resolve => {
+    const iframe = document.createElement("iframe");
+    iframe.onload = t.step_func(() => {
+      const { contentWindow } = iframe;
+      assert_equals(contentWindow.status, "");
```

Comment:
Should we assert more than status?

Reply:
Status suffices to reproduce the Chrome bug.